boards/arm/b_u585i_iot02a/pre_dt_board.cmake

```diff
@@ -0,0 +1,3 @@
+
+# Suppress "spi_bus_bridge" DTC warning
```

Comment:
I would use a descriptive comment like for other boards 

```suggestion
# SPI is implemented via octospi so node name isn't spi@...
```

Reply:
In fact, it would probably make sense to update all the other boards that have a similar issue (stm32h735g_disco, stm32h573i_dk, stm32h7b3i_dk, stm32l562e_dk, ...) but I am not sure what's an easy way to list them all.

Reply:
To clarify: my request for change is only about making the comment more useful, not about asking you to do the updates for all boards using octospi :)

Reply:
Instead of adding a `pre_dt_board.cmake` for all of these boards, it might make sense to suppress the warning on a global level for all boards? Is there a benefit for having this warning in certain cases?